arm/Microsoft.Network/azureFirewalls/deploy.bicep

```diff
@@ -118,16 +102,19 @@ param metricsToEnable array = [
   'AllMetrics'
 ]
 
-var diagnosticsLogsAzureFirewall = [for log in firewallLogsToEnable: {
-  category: log
-  enabled: true
-  retentionPolicy: {
-    enabled: true
-    days: diagnosticLogsRetentionInDays
+var ipConfigurations_var = [for ipConfiguration in ipConfigurations: {
+  name: ipConfiguration.name
+  properties: {
+    publicIPAddress: contains(ipConfiguration, 'publicIPAddressResourceId') ? {
+      id: ipConfiguration.publicIPAddressResourceId
+    } : null
+    subnet: contains(ipConfiguration, 'subnetResourceId') ? {
+      id: ipConfiguration.subnetResourceId
+    } : null
   }
 }]
 
-var diagnosticsLogsPublicIp = [for log in publicIPLogsToEnable: {
+var diagnosticsLogsAzureFirewall = [for log in firewallLogsToEnable: {
```

Comment:
Why the rename? Can't see any others.

Reply:
No renaming. The publicIP address incl. logs is not there anymore. The ones for the FW where already there and are untouched.

Reply:
Gotcha. BUT then let's call it 'diagnosticLogs' & 'logsToEnable' ;) Same for metrics

Reply:
done